server/templates/utility/site/flash_message.html

```diff
@@ -0,0 +1,14 @@
+{% with messages = get_flashed_messages() %}
+  <!-- code from poco.com which does the lifting for displaying flash messages for user feedback -->
+  <!-- See bootstrap.js docs here for how we get them to fade out http://v4-alpha.getbootstrap.com/components/alerts/ -->
```

Comment:
It works well for instant user feedback from what I've seen so far, so I don't care to mess with it anytime soon. And I'm not sure if Bootstrap will ever be fully yanked, so that may be a moot point.